.gitignore

```diff
@@ -5,3 +5,4 @@ target
 .idea	
 *.iml	
 .DS_Store
+/target/
```

Comment:
do we need `/target/`? we already have `target/`

Reply:
i didnt touch the gitignore?